src/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+  background-color: #000;
+  font-size: 16px;
+}
+
+.outer {
+  box-sizing: border-box;
+  width: 350px;
+  padding: 10px;
+  margin: 40px 20px;
+  background-color: #0090c2;
+  border: 1px solid #e95d49;
```

Comment:
if border is the same for all the containers, then why not create another class for all the containers and write border there?